### PR TITLE
adds monitoring infra stack

### PR DESCRIPTION
**Pull request by MehdiZonjy**
Creates aws resources required for moo monitoring stack
- RDS used to store grafana data
- S3 Bucket used by Thanos to store prometheus recordings

### CODE REVIEW

**Comment by aquila-droid** [comment]
Thanks for your pull request. Before we can look at it, you'll need to add a 'DCO signoff' to your commits.

:memo: **Please follow instructions in the [contributing guide](https://github.com/trainingrocket/stacks/blob/master/CONTRIBUTING.md) to update your commits with the DCO**

Full details of the Developer Certificate of Origin can be found at [developercertificate.org](https://developercertificate.org/).

**The list of commits missing DCO signoff**:

- [834f212](https://github.com/trainingrocket/stacks/commits/834f2128a63c2eb7135f8b8f444e0f452f3bf406) adds monitoring infra stack

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by aquila-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **rolobergmann**
You can assign the PR to them by writing `/assign @rolobergmann` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=trainingrocket%2Fstacks).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/trainingrocket/stacks/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by aquila-droid** [comment]
Thanks for your pull request. Before we can look at it, you'll need to add a 'DCO signoff' to your commits.

:memo: **Please follow instructions in the [contributing guide](https://github.com/trainingrocket/stacks/blob/master/CONTRIBUTING.md) to update your commits with the DCO**

Full details of the Developer Certificate of Origin can be found at [developercertificate.org](https://developercertificate.org/).

**The list of commits missing DCO signoff**:

- [834f212](https://github.com/trainingrocket/stacks/commits/834f2128a63c2eb7135f8b8f444e0f452f3bf406) adds monitoring infra stack
- [b26116a](https://github.com/trainingrocket/stacks/commits/b26116a66fed9a52f2054f50b55d23be76d6f7b6) fixes security group

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by aquila-droid** [comment]
LGTM label has been added.  <details>Git tree hash: 220b047a8d1f642fe5e668f17b7f4e662e88c4d1</details>

**Comment by aquila-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/trainingrocket/stacks/pull/10#pullrequestreview-323478841" title="Approved">faizshukri</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **rolobergmann**
You can assign the PR to them by writing `/assign @rolobergmann` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=trainingrocket%2Fstacks).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/trainingrocket/stacks/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: monitoring-stack/monitoring-stack.yaml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    Description: Private subnets which must contain 3 subnets for 3 AZs for DB
+
+  DBInstanceClass:
+    Type: String
+    Default: db.t3.small
+    AllowedValues:
+      - db.t3.small
+      - db.r5.medium
+
+  SnapshotIdentifier:
```
Comment (sanjevsunder) [inline monitoring-stack/monitoring-stack.yaml]: @MehdiZonjy You've defined SnapshotIdentifier parameter, but it's not used as a DBInstance property.

File: monitoring-stack/monitoring-stack.yaml [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+      Tags:
+        - Key: Product
+          Value: !Ref Product
+        - Key: Environment
+          Value: !Ref Environment
+        - Key: Team
+          Value: !Ref Team
+        - Key: Business
+          Value: !Ref Business
+    DeletionPolicy: Snapshot
```
Comment (sanjevsunder) [inline monitoring-stack/monitoring-stack.yaml]: Also, I highly recommend adding UpdateReplacePolicy : Snapshot

**Comment by yclian** [comment]
> You can assign the PR to them by writing /assign @rolobergmann in a comment when ready.

Is Rolo still being assigned to comment or we are breaking the policy?

cc: @sanjevsunder @dchui

**Pull request merged**